### DISCUSSION TITLE
Multiple search pages

### DISCUSSION THREAD

**Discussion by 13Groszy** [OrchardCMS/OrchardCore#15469] [Q&A]
I've got one website which split to 3 sub-sites (with multiple subpages) and I want to implement search page for each individual sub-site. E.g.
'sitename.com/site1/search' - search for everything under sitename.com/site1/*
'sitename.com/site2/search' - search for everything under sitename.com/site2/*
'sitename.com/site3/search' - search for everything under sitename.com/site3/*

Is there a chance to achieve that? At the moment I've only managed to create one search page 'sitename.com/search' which works fine, but display content from all 3 sub-sites.

Edit: I'm using Lucene Search, I've created unique search Indexes for each sub-site and one general which actual search page is using right now.

**Comment by Skrypt** [comment]
These 3 different sites would need to be different tenants.

**Reply by 13Groszy** [reply]
Is this a must? I was trying to avoid it...
So I'm not able to create custom search pages which will use different search Indexes?
In that case, creating more search indexes than one is pointless.

**Reply by Skrypt** [reply]
Well technically you can't because there is only one search index configuration set for the Search feature. The index would need to become a param of that search url or implied by that url/route. Yeah, maybe pointless for that Search feature but then you will need to adapt it for your needs.
